### PR TITLE
patch moto to respond w/ correct code and body when posting an upload

### PR DESCRIPTION
**Pull request by aschiopu**

### CODE REVIEW

File: moto/s3/responses.py [hunk 1]
```diff
@@ -212,6 +221,8 @@ def _bucket_response_post(self, request, bucket_name, headers):
                 form[k] = v
 
         key = form['key']
+        key = key.replace(
```
Comment (aschiopu) [inline moto/s3/responses.py:224]: this

File: moto/s3/responses.py [hunk 2]
```diff
@@ -223,12 +234,20 @@ def _bucket_response_post(self, request, bucket_name, headers):
         metadata = metadata_from_headers(form)
         new_key.set_metadata(metadata)
 
-        return 200, headers, ""
+        template = self.response_template(S3_POST_BUCKET_RESPONSE)
+        response = template.render(
```
Comment (aschiopu) [inline moto/s3/responses.py:238]: this

File: moto/s3/responses.py [hunk 3]
```diff
@@ -678,6 +703,15 @@ def _key_response_post(self, request, body, parsed_url, bucket_name, query, key_
 </CompleteMultipartUploadResult>
 """
 
+S3_POST_BUCKET_RESPONSE = """<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aschiopu) [inline moto/s3/responses.py:706]: this

**Comment by goosemo** [comment]
lgtm

**Pull request merged**